packages/tokens/properties/size/font.json

```diff
@@ -1,4 +1,64 @@
 {
+  "legacy": {
+    "size": {
+      "font": {
+        "h1": {
+          "value": "1.5rem",
+          "comment": "Font size for Heading 1 sized text"
+        },
+        "h2": {
+          "value": "1.125rem",
+          "comment": "Font size for Heading 2 sized text"
+        },
+        "h3": {
```

Comment:
_Why only define tokens up to H3?_

The H4 and H5 sizes are actually smaller than the body size, so it felt appropriate to leverage the "sm" and "xs" sizes instead